### PR TITLE
Add profile for kdiff3

### PR DESCRIPTION
**Pull request by Neo00001**

### CODE REVIEW

File: etc/profile-a-l/kdiff3.profile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+seccomp
+shell none
```
Comment (rusty-snake) [inline etc/profile-a-l/kdiff3.profile:38]: Can we add `tracelog`?

File: etc/profile-a-l/kdiff3.profile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+disable-mnt
+private-bin  kdiff3
+private-cache
+private-dev
```
Comment (rusty-snake) [inline etc/profile-a-l/kdiff3.profile:43]: Can we add `dbus-{user,system} {none,filter}`?

File: etc/profile-a-l/kdiff3.profile [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+net none
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+seccomp
```
Comment (rusty-snake) [inline etc/profile-a-l/kdiff3.profile:37]: Can we add `seccomp.block-secondary`.

**Review by rusty-snake — CHANGES_REQUESTED** [review]
kdiff3 has a very similar usage as meld. We should do the same filesystem restrictions as in [meld.profile](https://github.com/netblue30/firejail/blob/master/etc/profile-m-z/meld.profile).

Can you add all the 
```
# Uncomment the next line (or put it into your meld.local) if you don't need to compare files in …
```
comments to make this restrictions opt-in.

File: etc/profile-a-l/kdiff3.profile (hunk 3 above)
Comment (Neo00001) [reply etc/profile-a-l/kdiff3.profile:37]: yep.

File: etc/profile-a-l/kdiff3.profile (hunk 1 above)
Comment (Neo00001) [reply etc/profile-a-l/kdiff3.profile:38]: for some reason,I always tend to forget about `tracelog`.

File: etc/profile-a-l/kdiff3.profile (hunk 2 above)
Comment (Neo00001) [reply etc/profile-a-l/kdiff3.profile:43]: added

**Review by rusty-snake — APPROVED** [review]

**Comment by netblue30** [comment]
merged, thanks!

**Pull request merged**